release/RELEASE.md

```diff
@@ -353,9 +353,10 @@ we set `$VERSION` to be `2.4.0-snapshot.20220830.10494.0.4622de48`.
    1. Run `daml new myproject` to create a new project and switch to it using
       `cd myproject`.
    1. Run `daml start`.
-   1. Open your browser at `http://localhost:7500`, verify that you can log in as
+   1. ~~Open your browser at `http://localhost:7500`, verify that you can log in as
```

Comment:
The only way in which Navigator occurs in the release testing steps is to check what Navigator displays, so I'd just delete these.